.github/CONTRIBUTING.md

```diff
@@ -10,30 +10,52 @@ Be sure to follow our [code of conduct](https://github.com/ScratchAddons/Scratch
-5. [Create a new branch](https://docs.github.com/en/get-started/quickstart/contributing-to-projects#creating-a-branch-to-work-on) from the `master` branch. This makes things easier later on if you have multiple pull requests open at once or ever want to contribute again.
-6. [Commit and push](https://docs.github.com/en/get-started/quickstart/contributing-to-projects#making-and-pushing-changes) your changes to your new branch.
-7. If you haven't already, go to your fork on the website and enable workflows in Actions on your fork. This will enable an automation that formats (cleans up) your code if necessary.
-8. Go to the website to [create a pull request](https://github.com/ScratchAddons/ScratchAddons/compare) on the origin repository (ScratchAddons/ScratchAddons). There should be a notification giving you a shortcut to open a pull request. (If you don't see it, you can manually open a pull request by clicking "New pull request", then "Compare across forks" and selecting your fork and branch next to "head repository". Then click "Create pull request" to go to the next step.)
-9. Finally, fill out the form and click "Create pull request"! We will review your changes and talk about any further improvements if necessary.
+2. [Fork this repository](https://docs.github.com/en/get-started/quickstart/fork-a-repo) if you haven't already.
+3. [Clone your fork](https://docs.github.com/en/get-started/quickstart/fork-a-repo#cloning-your-forked-repository) and [load it into your browser](https://scratchaddons.com/docs/getting-started/installing/#from-source) so you can make and test your changes.
+4. [Create a new branch](https://docs.github.com/en/get-started/quickstart/contributing-to-projects#creating-a-branch-to-work-on), specifying `upstream/master` as the source branch:
+  ```shell
+  git checkout -b BRANCH-NAME upstream/master
+  ```
+  Branching makes things easier later on if you have multiple pull requests open at once or ever want to contribute again. You can always delete a branch.
+
+5. Make and test your changes.
+6. [Commit and push](https://docs.github.com/en/get-started/quickstart/contributing-to-projects#making-and-pushing-changes) your changes to the branch.
+7. Go to the website to create a pull request in the source repository (ScratchAddons/ScratchAddons). There should be a banner with a button to open a pull request.
+  ![Screenshot of the banner above the list of files.](https://docs.github.com/assets/cb-34106/mw-1440/images/help/pull_requests/pull-request-compare-pull-request.webp)
```

Comment:
The screenshot is fine but here's a quick tip: Firefox's built-in screenshot tool (<kbd>Ctrl</kbd> + <kbd>Shift</kbd> + <kbd>S</kbd>) let's you screenshot elements to get something that looks a little cleaner. I think it's also possible on chrome but it's more complicated.

Reply:
I actually took that screenshot from GitHub's documentation.

Reply:
I'm not sure if this URL would last forever. I think it would better to generate an URL for ourselves (uploading an image on this issue and copying the generated link is fine), and use that instead.

Reply:
Would this be okay?

https://github.com/github/docs/blob/78aee663ffc3f1f3eb1668efa0387e0febc97ede/assets/images/help/pull_requests/pull-request-compare-pull-request.png?raw=true

It's a raw link of the same image on a specific commit, and those are typically permanent.

```suggestion
  ![Screenshot of the banner above the list of files.](https://github.com/github/docs/blob/78aee663ffc3f1f3eb1668efa0387e0febc97ede/assets/images/help/pull_requests/pull-request-compare-pull-request.png?raw=true)
```

Reply:
That would do.